templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig

```diff
@@ -542,26 +577,47 @@
-                                            data-cy="r_publicParticipationPhase"
-                                            v-model="selectedPublicPhase"
-                                            name="r_publicParticipationPhase">
-                                            {% for externalPhase in templateVars.externalPhases %}
-                                                <option value="{{ externalPhase.key }}"{% if proceduresettings.publicParticipationPhase == externalPhase.key %} selected="selected"{% endif %} data-permissionset="{{ externalPhase.permissionset }}">
-                                                    {{ externalPhase.name }}
-                                                </option>
-                                            {% endfor %}
-                                        </select>
+                                        <div class="flex gap-2">
+                                            <div class="w-8-12">
+                                                <label class="inline-block mb-0.5" for="r_publicParticipationPhase">
+                                                    {{ "procedure.public.phase"|trans }}*
+                                                </label>
+                                                {% include '@DemosPlanCore/Extension/contextual_help.html.twig' with {
+                                                    helpText: 'text.procedure.edit.external.change_phase'|trans,
+                                                    cssClasses:'u-mt-0_125',
+                                                    showPlainHint: true
+                                                } %}
+                                                <select
+                                                    id="r_publicParticipationPhase"
+                                                    class="o-form__control-select u-11-of-12"
+                                                    data-cy="r_publicParticipationPhase"
+                                                    v-model="selectedPublicPhase"
+                                                    name="r_publicParticipationPhase">
+                                                    {% for externalPhase in templateVars.externalPhases %}
+                                                        <option value="{{ externalPhase.key }}"{% if proceduresettings.publicParticipationPhase == externalPhase.key %} selected="selected"{% endif %} data-permissionset="{{ externalPhase.permissionset }}">
+                                                            {{ externalPhase.name }}
+                                                        </option>
+                                                    {% endfor %}
+                                                </select>
+                                            </div>
+
+                                            {# @todo Permission Missing #}
```

Comment:
Is here also a permission missing?

Reply:
You are so right. Thanks for spotting that. Changed with [dcbc9ba](https://github.com/demos-europe/demosplan-core/pull/2834/commits/dcbc9ba5db8c74185491d242a11730875d744292)